src/Servant/Elm/Internal/Generate.hs

```diff
@@ -450,15 +465,63 @@ isEmptyType :: ElmOptions -> ElmDatatype -> Bool
 isEmptyType opts elmTypeExpr =
   elmTypeExpr `elem` emptyResponseElmTypes opts
 
+toStringSrc :: T.Text -> ElmOptions -> ElmDatatype -> Doc
+toStringSrc operator opts argType
+  -- Don't use "toString" on Elm Strings, otherwise we get extraneous quotes.
+  | isElmStringType opts argType = stext ""
+  | otherwise                    = stext $ toStringSrcTypes operator opts argType <> " " <> operator
+
 
 {- | Determines whether we call `toString` on URL captures and query params of
 this type in Elm.
 -}
 isElmStringType :: ElmOptions -> ElmDatatype -> Bool
+isElmStringType _ (ElmPrimitive (EList (ElmPrimitive EChar))) = True
```

Comment:
Is this saying that an elm `List Char` is an elm `String`? If so, how does that work?

Reply:
yes. don't know how it works, but it does 🙂